### ISSUE TITLE
Add the original "Coldbox" into the index.

### ISSUE THREAD

**Issue by Benjigif** [Incredimods/incredimods.github.io#4]
There's a "Coldbox Plus" but where is the original "Coldbox"? I don't see it.

**Comment by Benjigif** [comment]
Oops, sorry.

**Comment by Benjigif** [comment]
Hello?

**Comment by Clmsch2907** [comment]
It’s in the cold box plus

**Comment by Benjigif** [comment]
But V1 and V2's full version files are missing!

**Comment by Clmsch2907** [comment]
You are right but there are rv Mechanic and rv Bonfire (before the sound update)

**Comment by Benjigif** [comment]
I want to experience after the sound update.

**Comment by Benjigif** [comment]
@IncrediDexer can you please add the files of the full version of Mechanic to Mechanic Plus?

**Comment by Benjigif** [comment]
@IncrediDexer @IncrediDexer @IncrediDexer

**Comment by IncrediDexer** [comment]
I will move it to rems modding build.

**Comment by Benjigif** [comment]
@IncrediDexer sorry for the late response but thanks!
